modules/delegates.js

```diff
@@ -295,46 +295,46 @@ __private.checkDelegates = function (publicKey, votes, state, cb) {
  * @returns {setImmediateCallback}
  */
 __private.loadDelegates = function (cb) {
-	let secrets;
+	let secret;
 
 	if (library.config.forging.secret) {
 		if (Array.isArray(library.config.forging.secret)) {
```

Comment:
Array.isArray(library.config.forging.secret always one value remove if and secret.length